Dockerfile

```diff
@@ -23,7 +23,7 @@ FROM python:3.6
 RUN apt-get update && \
     apt-get install -y vim-tiny && \
     pip install --upgrade pip
-
+RUN pip install -e git://github.com/reanahub/reana-workflow-commons.git@master#egg=reana-workflow-commons
```

Comment:
The 2 pip install from github had to be moved here, before the installation of requirements-builder, as `reana-commons` and `reana-workflow-commons` need to be preset in the `setup.py`, and it fails otherwise.